Sources/Bugsnag/Bugsnag.swift

```diff
@@ -32,12 +32,13 @@ struct BugsnagEvent: Encodable {
         app: BugsnagApp,
         breadcrumbs: [BugsnagBreadcrumb],
         error: Error,
-        httpRequest: HTTPRequest,
+        httpRequest: HTTPRequest? = nil,
         metadata: [String: CustomDebugStringConvertible],
         payloadVersion: String,
         severity: Severity,
         stacktrace: BugsnagStacktrace,
-        userId: CustomStringConvertible?
+        userId: CustomStringConvertible?,
+        keyFilters: [String]
```

Comment:
Since `keyFilters` only makes sense when given a non-nil httpRequest I'm thinking if it would make more sense to replace both parameters with one optional `BugsnagRequest`. Note that the arguments here are (were) sorted alphabetically.